### PR TITLE
Ac 1183 certificate manager

### PR DESCRIPTION
**Pull request by ru-fu**

### CODE REVIEW

File: howto/manage/tls-for-appliance.md [hunk 1]
```diff
@@ -83,3 +90,147 @@ The `certbot` snap packages installed on your machine already set up a systemd t
...
+You can skip this step if you registered your domain name through Amazon Route 53.
+[/note]
+
+If you want to use a domain name that was not registered through Amazon Route 53, you must manually create a [public hosted zone](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/AboutHZWorkingWith.html) for it, and then make your domain use Amazon Route 53 as its DNS service. A public hosted zone contains routing information for the domain, and Amazon Route 53 uses it to determine to which machine it should route the traffic to your domain.
+
+In the case of the Anbox Cloud Appliance, we want the traffic routed to the AWS load balancer (which we will set up in [step 3](#step3)).
+
+1. To create a public hosted zone, follow the instructions in [Creating a public hosted zone](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingHostedZone.html) in the Amazon Route 53 documentation.
+
+   The following example shows a public hosted zone for the domain name `acappliance.tk`. The two DNS records are added automatically after creation:
```
Comment (adglkh) [inline howto/manage/tls-for-appliance.md:119]: Probably, it's worth mentioning that `acappliance.tk` was registered in advance from freenom.

**Review by adglkh — APPROVED** [review]
LGTM. two nitpicks.

File: howto/manage/tls-for-appliance.md [hunk 2]
```diff
@@ -83,3 +90,147 @@ The `certbot` snap packages installed on your machine already set up a systemd t
...
 [note type="information" status="Note"]The appliance will face a short downtime during the renewal of the SSL certificate but will come back online once the process is completed.[/note]
+
+<a name="aws-certificate-manager"></a>
+## Use the AWS Certificate Manager
+
+If you run the Anbox Cloud Appliance on AWS, you can use the [AWS Certificate Manager](https://aws.amazon.com/certificate-manager/) to provision and manage your TLS certificate.
+
+Before you start, make sure the following requirements are met:
+
+- You have a domain name for use with the Anbox Cloud Appliance registered, either through Amazon Route 53 (see [Register a new domain](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/domain-register.html) in the Amazon Route 53 documentation) or  through another domain provider.
```
[the remark this answers is not in the document]
Comment (morphis) [reply howto/manage/tls-for-appliance.md:101]: I think we should stay really abstract here and don't mention a specific domain (only something like `example.com`) or a DNS registrar.

File: howto/manage/tls-for-appliance.md (hunk 1 above)
Comment (morphis) [reply howto/manage/tls-for-appliance.md:119]: Is `acappliance.tk` a domain owned by your @adglkh ? If yes, we should not use it here.

Comment (ru-fu) [reply howto/manage/tls-for-appliance.md:119]: It is used in all the screenshots - we can't really provide screenshots without a domain name, and editing it to something like example.com won't look very good I fear.

File: howto/manage/tls-for-appliance.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ru-fu) [reply howto/manage/tls-for-appliance.md:101]: I agree that we shouldn't mention freedom - users should decide where they want to register their domain.
I think we need to give a specific domain name though, see my comment below.

**Review by adglkh — APPROVED** [review]
LGTM.
Thanks!

**Review by morphis — APPROVED** [review]
LGTM, thanks!

**Pull request merged**